ISISNeutronMuon/MDANSE
# Fix ase converter

**Description of work**
The ASE converter is now able to take an ASE .traj file and convert it to the .MDT format.

**Fixes**
- changed the ASE converter code
- added a sample trajectory to the tests Data directory
- added a corresponding test to the test_converters.py file

**To test**
Please confirm that the tests pass. Optionally try to convert and load the sample trajectory in the GUI.
